fortify-issue-manager/src/main/java/com/microfocus/security/automation/fortify/issue/tracker/OctaneTracker.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.microfocus.security.automation.fortify.issue.tracker;
+
+import com.google.common.net.UrlEscapers;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.microfocus.security.automation.fortify.issue.manager.BugTracker;
+import com.microfocus.security.automation.fortify.issue.manager.models.Vulnerability;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public final class OctaneTracker implements BugTracker {
+    private final static JsonParser parser = new JsonParser();
+    private final String apiUrl;
+    private final String browseUrl;
+    private final String defectUrl;
+    private final OctaneTrackerClient client;
+
+    public OctaneTracker(final OctaneBugTrackerSettings bugTrackerSettings) {
```

Comment:
Either `OctaneBugTrackerSettings` should be public or `OctaneTracker` shouldn't be.  (Probably the latter.)